src/main/scala/com/thirdship/libunit/units/MetricTSUnit.scala

```diff
@@ -0,0 +1,52 @@
+package com.thirdship.libunit.units
+
+import com.thirdship.libunit.utils.{ExactString, FuzzyString, WordString}
+import com.thirdship.libunit.utils.Helpers._
+import com.thirdship.libunit._
+
+class MetricTSUnit(unit: String, parseList: List[FuzzyString]) {
```

Comment:
hmm, it would be nice if this class were called helpers or something down that line, and if it was an object. For example:

```
private val compressedParseMap = Map(
...
    "jiffy".i   -> List()
) + MetricHelpers.compressedParseMap(baseUnit,baseParseList)

private val edges = List(
    new ScalarConversionEdge(baseUnit,  "jiffy",    3e-24,      1)
) + MetricHelpers.conversionsEdges(baseUnit,baseParseList)
```

The reason behind this is that it might be easier to read. That said, if it was implemented as a helper, for ConvertibleTSUnitData and AStarConvertibleTSUnitData like
new `AStarConvertibleTSUnitData(baseUnit, "Time", compressedParseMap, edges, metricUnits=List(baseUnit))`
or, `AStarConvertibleTSUnitData(baseUnit, "Time", compressedParseMap, edges).createMetricUnits(List(baseUnit))`

Feel that the last one is the best as it would not require any declarations, and will allow the code to remain nice and neat in the base class for AStarConvertibleTSUnitData and ConvertibleTSUnitData without requiring a code change.